etc/skydive.yml.default

```diff
@@ -229,6 +230,9 @@ agent:
       run_path:
         # - /var/run/runc
 
+    vpp:
+      # connect: ""
```

Comment:
could be nice to give a real example

Reply:
It's most likely because they introduce prefix in api-segment

https://wiki.fd.io/view/VPP/Command-line_Arguments#.22api-segment.22_parameters

But I agree, in most case it's useless